common/source/docs/common-rc-transmitter-flight-mode-configuration.rst

```diff
@@ -90,7 +93,9 @@ parameter. DEfault is channel 8.
 [site wiki="rover"]
 On Rover this is configurable using the :ref:`MODE_CH <rover:MODE_CH>` parameter. Default is channel 8,
 [/site]
-
+[site wiki="sub"]
+By default, :ref:`FLTMODE_CH<FLTMODE_CH>` is set to "0", disabling the ability to select modes via RC, but can be set to any otherwise unused available RC channel.
```

Comment:
It would be nice to link this to [docs of the default channel configuration](https://github.com/ArduPilot/ardupilot_wiki/blob/master/sub/source/docs/sub-hardware.rst?plain=1#L26), so it's clearer which channels are likely to be unused.

Reply:
I expect when using RC several other changes will be made to RC inputs....